sdk/enoki/src/EnokiClient/index.ts

```diff
@@ -134,6 +140,40 @@ export class EnokiClient {
 		);
 	}
 
+	getSubnames(input: GetSubnamesApiInput) {
+		return this.#fetch<GetSubnamesApiResponse>('subnames', {
+			method: 'GET',
+			body: JSON.stringify({
+				address: input.address,
+				network: input.network,
+				domain: input.domain,
+			}),
+		});
+	}
+
+	createSubname(input: CreateSubnameApiInput) {
```

Comment:
How do we pass the jwt here - for the case we are using the public api key?

Reply:
good catch, I'll get that added

Reply:
added a jwt input